Header header
geometry_msgs/Twist twist
geometry_msgs/Pose pose
float64 propeller_mass